gadgets/top_blockio/gadget.yaml

```diff
@@ -0,0 +1,50 @@
+name: top blockio
+description: Periodically report input/output block device activity. This gadget requires
```

Comment:
I think "This gadget requires Kernel Version 6.5+" doesn't belong to the description.